cluster-autoscaler/core/scaledown/planner/planner_test.go

```diff
@@ -1035,6 +1035,119 @@ func TestNodesToDelete(t *testing.T) {
 	}
 }
 
+func TestLongestUnprocessedNodeScaleDownTime(t *testing.T) {
+	type testCase struct {
+		name                         string
+		unprocessedNodes             [][]string
+		wantLongestScaleDownEvalTime []time.Duration
+	}
+	start := time.Now()
+	testCases := []testCase{
```

Comment:
Can you add a test case when the set of skipped nodes is different every time? I would expect the metric not to grow in this case.

Reply:
Similarly as in the other comment - this sounds like an artifact of the current implementation, rather than desired behavior. Why would we want the metric to be different for a single unprocessed node based on whether previous iteration had any unprocessed nodes or not?